blues/moves.py

```diff
@@ -177,70 +178,304 @@ def move(self, context):
+            particle_pos = np.asarray(context_pos._value)[self.basis_particles]*unit.nanometers
+            self.calculateProperties()
+            self.center_of_mass = self.getCenterOfMass(lig_pos, self.masses)
+            #get particle positions
+            new_coord = self._findNewCoord(particle_pos[0], particle_pos[1], particle_pos[2], self.center_of_mass)
+            #keep this in for now to check code is correct
+            #old_coord should be equal to com
+            old_coord = self._findOldCoord(particle_pos[0], particle_pos[1], particle_pos[2], new_coord)
+            n_dartboard.append(new_coord)
+            dartboard.append(old_coord)
+
+        self.n_dartboard = n_dartboard
+        self.dartboard = dartboard
+
+    def dartsFromMDTraj(self, system, file_list, topology=None):
+        """
+        Used to setup darts from a generic coordinate file, through MDtraj using the basis_particles to define
+        new basis vectors, which allows dart centers to remain consistant through a simulation.
+        This adds to the self.n_dartboard, which defines the centers used for smart darting.
+        Arguments
+        ---------
+        system: simtk.openmm.system
+            Openmm System corresponding to the system to smart dart.
+        file_list: list of str
+            List containing coordinate files of the system for smart darting.
+        basis_particles: list of 3 ints
+            Specifies the 3 indices of particles whose coordinates will be used
+            as basis vectors. If None is specified, uses those found in basis particles.
+            If None uses self.basis_particles instead.
+        Returns
+        -------
+        n_dartboard: list of 1x3 np.arrays
+            Center of mass coordinates of atom_indices particles in new basis set.
+        """
+        atom_indices = self.atom_indices
+        basis_particles = self.basis_particles
+        n_dartboard = []
+        dartboard = []
+        for md_file in file_list:
+            if topology == None:
+                temp_md = md.load(md_file)
+            else:
+                temp_md = md.load(md_file, top=topology)
+            context_pos = temp_md.openmm_positions(0)
+            context_pos = np.asarray(context_pos._value)*unit.nanometers
+            total_mass, mass_list = self.get_particle_masses(system, set_self=False, atom_indices=atom_indices)
+            com = self.calculate_com(pos_state=context_pos,
+                                    total_mass=total_mass,
+                                    mass_list=mass_list,
+                                    atom_indices=atom_indices)
+            #get particle positions
+            particle_pos = []
+            for particle in basis_particles:
+                particle_pos.append(context_pos[particle])
+            new_coord = self._findNewCoord(particle_pos[0], particle_pos[1], particle_pos[2], com)
+            #keep this in for now to check code is correct
```

Comment:
If this is a check, shouldn't you be, er, checking it? Like an assertion? Or is it checked elsewhere?